src/graphql/search/index.ts

```diff
@@ -21,6 +21,12 @@ const queryResolvers: QueryResolvers = {
 
 const resolversMap: Resolvers = {
   Query: queryResolvers,
+  // resolveType is used by GraphQL to handle inline fragments that determine which fields must be returned for this type
+  MapLayerLegendItem: {
+    __resolveType(obj: any) {
```

Comment:
We should know the type of object passed into here is a `MapLayer | LegendItem` right?

Reply:
Yes true, but TS already know what the type will be. So this can be removed at all